tests/tests/swfs/avm1/movieclip_default_state/test.as

```diff
@@ -0,0 +1,83 @@
+
+
+/*
+ * This returns all properties and the results of some getter functions of a MovieClip with their respective
+ * names in a two-dimensional array.
+ */
+function getMcPropsArray(mc:MovieClip) {
+	var mcProps = [];
+	var simplePropNames = ["_accProps", "_alpha", "_currentframe", "_droptarget", "_focusrect", "_framesloaded",
+		"_height", "_highquality", "_lockroot", "_name", "_parent", "_quality", "_rotation", "_soundbuftime",
+		"_target", "_totalframes", "_visible", "_width", "_x", "_xmouse", "_xscale", "_y", "_ymouse", "_yscale",
+		"blendMode", "cacheAsBitmap", "enabled", "filters", "filters.length", "focusEnabled", "forceSmoothing",
+		"hitArea", "menu", "opaqueBackground", "scale9Grid", "scrollRect", "tabChildren", "tabEnabled", "tabIndex",
+		"trackAsMenu", "transform.colorTransform", "transform.concatenatedColorTransform", "transform.matrix",
+		"transform.concatenatedMatrix", "transform.pixelBounds", "useHandCursor"];
+	for (var simplePropIterator = 0; simplePropIterator < simplePropNames.length; simplePropIterator++) {
+		var simplePropName = simplePropNames[simplePropIterator];
+		var simplePropValue = eval("mc." + simplePropName);
+		mcProps.push([simplePropName, simplePropValue]);
+	}
+
+	var url = unescape(mc._url);
+	if (url.indexOf("file:///") == 0) {
+		var urlSplit = url.split("/");
+		mcProps.push(["_url", urlSplit[urlSplit.length - 2] + "/" + urlSplit[urlSplit.length - 1]]);
+	} else {
+		mcProps.push(["_url", url]);
+	}
+
+	var getBoundsThis = mc.getBounds(this);
+	mcProps.push(["getBounds(this).xMin", getBoundsThis.xMin]);
+	mcProps.push(["getBounds(this).xMax", getBoundsThis.xMax]);
+	mcProps.push(["getBounds(this).yMin", getBoundsThis.yMin]);
+	mcProps.push(["getBounds(this).yMax", getBoundsThis.yMax]);
+	var getBoundsMc = mc.getBounds(mc);
+	mcProps.push(["getBounds(mc).xMin", getBoundsMc.xMin]);
+	mcProps.push(["getBounds(mc).xMax", getBoundsMc.xMax]);
+	mcProps.push(["getBounds(mc).yMin", getBoundsMc.yMin]);
+	mcProps.push(["getBounds(mc).yMax", getBoundsMc.yMax]);
+	mcProps.push(["getBytesLoaded()", mc.getBytesLoaded()]);
+	mcProps.push(["getBytesTotal()", mc.getBytesTotal()]);
+	mcProps.push(["getDepth()", mc.getDepth()]);
+	mcProps.push(["getInstanceAtDepth(0)", mc.getInstanceAtDepth(0)]);
+	mcProps.push(["getNextHighestDepth()", mc.getNextHighestDepth()]);
+	var getRectThis = mc.getBounds(this);
+	mcProps.push(["getRect(this).xMin", getRectThis.xMin]);
+	mcProps.push(["getRect(this).xMax", getRectThis.xMax]);
+	mcProps.push(["getRect(this).yMin", getRectThis.yMin]);
+	mcProps.push(["getRect(this).yMax", getRectThis.yMax]);
+	var getRectMc = mc.getBounds(mc);
+	mcProps.push(["getRect(mc).xMin", getRectMc.xMin]);
+	mcProps.push(["getRect(mc).xMax", getRectMc.xMax]);
+	mcProps.push(["getRect(mc).yMin", getRectMc.yMin]);
+	mcProps.push(["getRect(mc).yMax", getRectMc.yMax]);
+	mcProps.push(["getSWFVersion()", mc.getSWFVersion()]);
+	mcProps.push(["getTextSnapshot().getCount()", mc.getTextSnapshot().getCount()]);
```

Comment:
You could probably just comment out this line, as well as the filters line, until they get imlemented, and add a TODO notice.

Reply:
Tbh yeah, it probably makes most sense to do that for now. I just hope I remember to add it back again if they get implemented.